services/worker-manager/src/api.js

```diff
@@ -173,34 +179,6 @@ builder.declare({
   res.reply(workerPool.serializable());
 });
 
-builder.declare({
```

Comment:
I sorta wonder if we should keep this endpoint and have it just set the `providerId` to `"null"` itself?

Reply:
I opted to get rid of the endpoint since it's not really deleting anymore -- I think folks might be surprised when after "deleting" a worker pool that pool still appears in the list.